package.json

```diff
@@ -44,5 +45,8 @@
     "rollup-plugin-eslint": "^7.0.0",
     "rollup-plugin-sourcemaps": "^0.6.3",
     "rollup-plugin-terser": "^7.0.2"
+  },
+  "dependencies": {
+    "regenerator-runtime": "^0.14.1"
   }
```

Comment:
why we adding this dependency?

Reply:
Since for fetch we are using async await and babel core version is 7.11.4 we need this 
Refer - https://babeljs.io/docs/babel-polyfill